opensrp-app/src/main/java/org/smartregister/cursoradapter/SecuredNativeSmartRegisterCursorAdapterFragment.java

```diff
@@ -399,9 +403,14 @@ private int getTotalcount() {
     }
 
     public void refresh() {
+        int textSize = getResources().getDimensionPixelSize(R.dimen.pagination_page_info_size);
+        int startIndex = getResources().getString(R.string.str_page_info).indexOf("{");
         pageInfoView.setText(
                 format(getResources().getString(R.string.str_page_info), (getCurrentPageCount()),
                         getTotalcount()));
+        SpannableString span = new SpannableString(pageInfoView.getText());
+        span.setSpan(new AbsoluteSizeSpan(textSize), startIndex, pageInfoView.getText().length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+        pageInfoView.setText(span);
```

Comment:
Any reason for this extra logic?

Reply:
just for reduce the page info like `1 of 3`

Reply:
I think the requirement was to allow the developer to override a method that determines how the pagination information is displayed? 

The size shouldn't be reduced in core.

Reply:
fixed.